src/plugins/validation/2and3/semantic-validators/info.js

```diff
@@ -0,0 +1,40 @@
+// Assertation 1:
+// check if info exists
+
+// Assertation 2:
+// making sure that the required version and title are defined properly
+module.exports.validate = function({ jsSpec }) {
+  const errors = [];
+  const warnings = [];
+  const info = jsSpec.info;
+  if (!info) {
+    errors.push({
+      path: ['info'],
+      message: 'Missing info object for api defintion'
+    });
+  } else if (typeof info != 'object') {
+    errors.push({
+      path: ['info'],
+      message: 'Info is an obect and must have properties'
+    });
+  } else {
+    const title = jsSpec.info.title;
+    const hasTitle =
+      typeof title === 'string' && title.toString().trim().length > 0;
+    const version = jsSpec.info.version;
+    const hasVersion =
+      typeof version === 'string' && version.toString().trim().length > 0;
+    if (!hasTitle) {
+      errors.push({
+        path: ['info', 'title'],
+        message: ' Info object is missng the title field'
```

Comment:
Let's use an error message of "\`info\` object must have a string-type \`title\` field"

Most of the error messages have the format of stating the requirement rather than the problem, so I think it's good to try for that when we can. You can use the same message for the version check. Also they should not start with a space